pom.xml

```diff
@@ -24,7 +24,7 @@
     <version>1.1.2-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.277.1</jenkins.version>
+        <jenkins.version>2.319</jenkins.version>
```

Comment:
How will this Jenkins version bring new dependencies?

Reply:
@jragula-zs It will download org.jenkins-ci.main:jenkins-core with this version i.e. 2.319